src/bindeps_conda.jl

```diff
@@ -1,57 +1,46 @@
 # This file contains the necessary ingredients to create a PackageManager for BinDeps
 using BinDeps
 
-type Manager <: BinDeps.PackageManager
+type ManagerType{T} <: BinDeps.PackageManager
     packages::Vector{Compat.ASCIIString}
 end
 
-function Base.show(io::IO, manager::Manager)
+"Manager for root environment"
+Manager = ManagerType{Symbol(PREFIX)}
+
+function Base.show{T}(io::IO, manager::ManagerType{T})
     write(io, "Conda packages: ", join(manager.packages, ", "))
```

Comment:
We already have `Conda packages: symengine==0.2.0 at /home/isuru/.julia/v0.4/Conda/deps/usr/lib/libsymengine.so`. Printing `T` would print the path `/home/isuru/.julia/v0.4/Conda/deps/usr/` which is redundant.